types/types.go

```diff
@@ -24,58 +23,19 @@ import (
 	"github.com/prometheus/common/model"
 )
 
-type AlertState uint8
+type AlertState string
 
 const (
-	AlertStateUnprocessed AlertState = iota
-	AlertStateActive
-	AlertStateSuppressed
+	AlertStateUnprocessed AlertState = "unprocessed"
```

Comment:
With pretty high confidence, I'd say no here. This is in probably in the low nanosecond range.